contrib/Docker_Grafana_Blueflood/docker-entrypoint.sh

```diff
@@ -37,6 +37,8 @@ blueflood:
   apikey: $RAX_APIKEY                     
   authentication_module: rax_auth
   authentication_class: BluefloodAuth
+  enable_submetrics: true
+  submetric_aliases: { "_sum": 'sum' }
```

Comment:
I am only thinking about the uptime reporting.  I am still testing it out to see how it works out.

We can add others when we find good use case for them.